src/Forms/Main.cs

```diff
@@ -7402,18 +7408,43 @@ private void ContextMenuStripListviewOpening(object sender, System.ComponentMode
                     toolStripMenuItemWebVTT.DropDownItems.Add(_language.Menu.ContextMenu.WebVTTRemoveVoices, null, WebVTTRemoveVoices);
                 }
             }
-            else if ((format.Name == "Nuendo" && SubtitleListview1.SelectedItems.Count > 0))

```

Comment:
comparing format.Name with `Nuendo` would always return false, there is no subtitle format named `Nuendo` but `Csv Nuendo`